pkg/capabilities/consensus/ocr3/types/aggregator.go

```diff
@@ -21,13 +23,24 @@ type Metadata struct {
 	ReportID         string //  2 hex bytes
 }
 
+// the contract requires exactly 10 bytes for the workflow name
+// the json schema allows for a variable length string <= len(10)
+// pad with trailing spaces to meet the contract requirements
+func (m *Metadata) padWorkflowName() {
+	if len(m.WorkflowName) < 10 {
+		suffix := strings.Repeat(" ", 10-len(m.WorkflowName))
```

Comment:
Actually, I think it might make sense to allow spaces in the name 🙈

Reply:
we don't allow spaces in any of the other identifiers (like ref). it seems easier and consistent to keep it simple here